### ISSUE TITLE
Evaluation - 18f.gsa.gov

### ISSUE THREAD

**Issue by AKM399** [GSA/EDX#173]
Website Evaluation of 18f.gsa.gov - Julie Strothman

- [x] Set appointment
- [x] view dashboard
- [x] View website
- [x] load notes to github
- [x] load additional lighthouse data
- [x] meet with website manager
- [x] include notes
- [ ] Report

**Comment by AKM399** [comment]
1/28 4-5 with @AKM399 and @anatecture

**Comment by AKM399** [comment]
dashboard - https://drive.google.com/file/d/1HlNR9MMX18SNN-F9pbCm7JZHb8S_1xQk/view?usp=sharing

**Comment by AKM399** [comment]
targeted at government.   Hard to discern purpose by first glance.  

missing some very low hanging fruit 508 wise https://wave.webaim.org/report#/18f.gsa.gov

good not great seo https://freetools.seobility.net/en/seocheck/18f.gsa.gov

generally clean design - cant see anything that doesnt belong

**Comment by AKM399** [comment]
Additional analytics
https://drive.google.com/drive/folders/1bZ7YWDvHCqUB2tT6yfPTBSFj89lYSr5R?usp=sharing

**Comment by anatecture** [comment]
## 1/28 Qual research interview
### Summary
Do not know who customers are. Are currently doing this research in response to a TTS DC request. Sites are informational, have uses as reference and recruiting assets. Mostly targeted to 18F folks and folks from similar, start-up and tech world backgrounds. Group wants to manage these sites as a single product. 

[Full notes from interview.](https://docs.google.com/document/d/1DgLme-kJZRtjX6j_rUEx6ah-1uXQkF8LScm8YMBIX5g/edit)

## tl/dr:
We will circle back with them in 2 months to see the results of their current research. 

## Immediate To-Dos:
- [ ] Connect JS with @RachelF to do a mark up of sites for generalized plain language. 

Interview data
Scan data

**Comment by AKM399** [comment]
updated analytics
https://docs.google.com/spreadsheets/d/1DTavuY6QI5xM4goDIAenpbrPWsea3UqGoWXwph-7xEw/edit?usp=sharing

**Comment by AKM399** [comment]
report ready for review:  https://docs.google.com/document/d/1kkUgHe4oGC7K79yFGlX7skgBbJqS__KCy3gEMaErSfk/edit?usp=sharing
@anatecture @RachelF @hatfieldjm4 @mvogelgesang if anyone is able to pick up the review on this one.
